packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -6,11 +6,17 @@ const uniq = (arr) => [...new Set(arr)];
 exports.onPreBootstrap = ({ reporter, store }) => {
   const { program } = store.getState();
   const imagePath = path.join(program.directory, 'src/images');
+  const announcementsPath = path.join(program.directory, 'src/announcements');
 
   if (!fs.existsSync(imagePath)) {
     reporter.info('creating the images directory');
     fs.mkdirSync(imagePath, { recursive: true });
   }
+
+  if (!fs.existsSync(announcementsPath)) {
```

Comment:
what is this checking here?

Reply:
This checks if an announcements directory exists in the filesystem. If not, it will create an `announcements` directory.